go/appencryption/pkg/persistence/dynamodb_test.go

```diff
@@ -339,10 +340,23 @@ func (suite *DynamoDBSuite) TestDynamoDBMetastore_Store_WithFailureShouldReturnE
 
 func (suite *DynamoDBSuite) TestDynamoDBMetastore_WithDynamoDBRegionSuffix() {
```

Comment:
unrelated to this PR but would it be better if we rename `WithDynamoDBRegionSuffix` to `WithKeySuffix`. Since this is in the `dynamodb.go` file, the DynamoDB in the name seems redundant, and since we call the variable keySuffix, `WithKeySuffix` seems to be more intuitive?

Reply:
I'd prefer to keep this as is. This is the `dynamodb.go` file, sure, but the file is part of the `persistence` package, so it's not really redundant. Renaming would mean users would reference the function as `persistence.WithKeySuffix(true)` which is ambiguous compared to the current `persistence.WithDynamoDBRegionSuffix(true)`.